src/currencyRatesFileGen.js

```diff
@@ -0,0 +1,59 @@
+// prebidCurrencyRatesFileGenerator
+// loads currency rates from api.fixer.io and creates a JSON representation in S3
+// revision:
+var http = require('http');
+var AWS = require('aws-sdk');
```

Comment:
probably worth adding a `package.json` so this runs as a stand alone project (I'm assuming AWS does this automatically)